src/Fhir.Anonymizer.Tool/FilesAnonymizerForNdJsonFormatResource.cs

```diff
@@ -96,13 +114,24 @@ public async Task AnonymizeAsync()
                         Console.WriteLine($"[{stopWatch.Elapsed.ToString()}][tid:{args.CurrentThreadId}]: {completedCount} Process completed. {failedCount} Process failed. {consumeCompletedCount} Consume completed.");
                     };
 
-                    await executor.ExecuteAsync(CancellationToken.None, false, progress).ConfigureAwait(false);
+                    await executor.ExecuteAsync(CancellationToken.None, true, progress).ConfigureAwait(false);
```

Comment:
what is "true" here for?

Reply:
it is for "breakOnAnonymizationException". Now we have resume, so we do not need skip on failure.

public async Task ExecuteAsync(CancellationToken cancellationToken, bool breakOnAnonymizationException = false, IProgress<BatchAnonymizeProgressDetail> progress = null)